src/components/Header/Polling.tsx

```diff
@@ -81,6 +106,10 @@ export default function Polling() {
   const [isHover, setIsHover] = useState(false)
   const machineTime = useMachineTimeMs(NETWORK_HEALTH_CHECK_MS)
   const blockTime = useCurrentBlockTimestamp()
+  const theme = useTheme()
+
+  const ethGasPrice = useGasPrice()
+  const priceGwei = ethGasPrice ? JSBI.divide(ethGasPrice, JSBI.BigInt(1000000000)) : undefined
```

Comment:
```suggestion
  const priceGwei = ethGasPrice ? JSBI.divide(ethGasPrice, JSBI.BigInt(1_000_000_000)) : undefined
```

Reply:
should confirm all browsers we target support this or that it gets removed at compilation

Reply:
could also use `1e9` for readability?